### PR TITLE
New Init and Start Menu scenes

### PR DESCRIPTION
**Pull request by Youenn-Bouglouan**
Created _init_scene_ that loads first at game startup and contains all objects that should persist across scenes (DevHelper, in-game console, Fps counter...).

Once _init_scene_ has loaded, it auto-loads _start_menu_ scene. From start_menu scene player can load one of the 2 levels available by clicking on the buttons.
From level scenes, pressing Escape will get the player back to _start_menu_.

Minor changes:
- changed UnityConsole notification processor to ignore _Trace_, _Info _and _Gameplay _severity levels.
- removed old, obsolete prefabs.
- cleaned up _Level_01_ scene.

### CODE REVIEW

File: Assets/Scripts/Helpers/SceneUtilities/AutoLoadStartMenu.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+        void Update()
+        {
+            var levelLoader = GetComponent<LevelLoader>();
+            if (levelLoader == null)
+                DevHelper.NotificationPublisher.Publish("[AutoLoadStartMenu] Could not find LevelLoader!", Severity.Error,
+                    "Make sure LevelLoader script is added to the current GameObject");
+            else
+                DevHelper.NotificationPublisher.Publish("Game initialization complete, proceeding to Start Menu.");
+
+            levelLoader.GoToStartMenu();
```
Comment (MrPeKa) [inline Assets/Scripts/Helpers/SceneUtilities/AutoLoadStartMenu.cs]: Maybe attach this to upper 'else' :)

**Comment by MrPeKa** [comment]
Excellent!

File: Assets/Scripts/Helpers/SceneUtilities/AutoLoadStartMenu.cs (hunk 1 above)
Comment (Youenn-Bouglouan) [inline Assets/Scripts/Helpers/SceneUtilities/AutoLoadStartMenu.cs]: Good remark! But in that case I want it to fail either way (we will get both the notification for clarity and the exception).

Comment (MrPeKa) [inline Assets/Scripts/Helpers/SceneUtilities/AutoLoadStartMenu.cs]: Sure, we can get exception or we can just stop program at if, it's up to you. Anyway PR accepted.

**Pull request merged**